node-template/runtime/src/lib.rs

```diff
@@ -189,6 +189,7 @@ parameter_types! {
 impl babe::Trait for Runtime {
 	type EpochDuration = EpochDuration;
 	type ExpectedBlockTime = ExpectedBlockTime;
+	type EpochChangeTrigger = babe::SameAuthoritiesForever;
```

Comment:
Maybe put a comment here for anyone that uses node-template to be aware and change this as soon as they want to do anything more than merely _just playing around_.